shared/crossCompilation/rust.md

```diff
@@ -0,0 +1,50 @@
+#### Setting up {{ $language.name }} cross-compilation
```

Comment:
Probably need a note/warning that this setup is for debian/ubuntu linux and won't work for OSX or Windows :( and ultimately we need to have a 3rd dimension for development machine OS